core/src/test/scala/unit/kafka/zk/KafkaZkClientTest.scala

```diff
@@ -70,30 +72,50 @@ class KafkaZkClientTest extends ZooKeeperTestHarness {
 
     zkClient.createRecursive("/create/some/random/long/path")
     assertTrue(zkClient.pathExists("/create/some/random/long/path"))
-    zkClient.createRecursive("/create/some/random/long/path") // no errors if path already exists
+    zkClient.createRecursive("/create/some/random/long/path", throwIfPathExists = false) // no errors if path already exists
 
     intercept[IllegalArgumentException](zkClient.createRecursive("create-invalid-path"))
   }
 
   @Test
-  def testGetTopicPartitionCount() {
-    val topic = "mytest"
+  def testTopicAssignmentMethods() {
```

Comment:
Saw the following unit test failure.

kafka.zk.KafkaZkClientTest > testTopicAssignmentMethods FAILED
    java.lang.AssertionError: expected:<List(topic1, topic2)> but was:<Buffer(topic2, topic1)>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at kafka.zk.KafkaZkClientTest.testTopicAssignmentMethods(KafkaZkClientTest.scala:117)